apps/graphql/src/apps/itinerary/types/inputs/ItinerariesCommonTypesInput.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import {
+  GraphQLInputObjectType,
+  GraphQLEnumType,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from 'graphql';
+import { GraphQLDate } from 'graphql-iso-date';
+
+import PassengersInput from './PassengersInput';
+
+// Define the common GraphQL types accross the Search Queries schema
+
+const DateRange = new GraphQLInputObjectType({
```

Comment:
AFAIK we create one GraphQL type per one file. Could you separate it to more files, please?

Reply:
You mean separating that file into one file per type?

Reply:
Yes

Reply:
@jvaclavik done some refactor of the input folder into `enums` and `common`